plugins/modules/route53_crrs.py

```diff
@@ -0,0 +1,504 @@
+#!/usr/bin/python
+# This file is part of Ansible
+# GNU General Public License v3.0+ (see COPYING or
+# https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+from ansible_collections.amazon.aws.plugins.module_utils.aws.core import (
+    AnsibleAWSModule,
+)
+
+__metaclass__ = type
+
+DOCUMENTATION = '''
+module: route53_crrs
+short_description: create, update or delete route53 resource record sets.
+description: create, update or delete route53 resource record sets.
+requirements: [boto3, botocore, python>=3.5]
+options:
+  action:
+    description: >
```

Comment:
```suggestion
    description:
```
You don't want the description to start with a `-`. The description should be a string (if you need one paragraph), or a YAML list (if you need one or multiple paragraphs).